cfme/tests/services/test_service_catalogs.py

```diff
@@ -236,3 +238,54 @@ def test_order_service_after_deleting_provider(appliance, provider, setup_provid
     assert view.details.request_details.get_text_of("Request State") == "Finished"
     last_msg = f"Error: Source Template/Vm with id [{template_id}] has no EMS, unable to provision"
     assert view.details.request_details.get_text_of("Last Message") == last_msg
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1380197])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize("import_data", [DatastoreImport("bz_1380197.zip", "bz_1380197", None)],
+                         ids=["datastore"])
+def test_service_catalog_email_request_approval_pending(request, appliance, import_datastore,
+                                                        generic_catalog_item):
+    """
+           Test to check is there have pending email approval status or not in log file after
+            ordering service catalog
+    Bugzilla:
+      1380197
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        testtype: functional
+        initialEstimate: 1/6h
+        tags: service
+        testSteps:
+            1. order service catalog
+        expectedResults:
+            1. pending email approval status message should be there in automation.log file
+    """
+
+    service_catalogs = ServiceCatalogs(
+        appliance, catalog=generic_catalog_item.catalog, name=generic_catalog_item.name,
+    )
+
+    with LogValidator(
+        "/var/www/miq/vmdb/log/automation.log",
+        matched_patterns=[
+            (
+                r'INFO.*Invoking \[builtin\] method \[/\$/Object/send_email\] with '
+                r'inputs.*Service Request from  Pending Approval'
+            )
+        ],
+        hostname=appliance.hostname,
+    ).waiting(timeout=900):
+        service_request = service_catalogs.order(wait_for_view=30)
+
+        # Remove Request
+        request_description = "Provisioning Service [{name}] from [{name}]".format(
+            name=generic_catalog_item.name
+        )
+        service_request = appliance.collections.requests.instantiate(
+            description=request_description, partial_check=True
+        )
+
+    request.addfinalizer(lambda: service_request.remove_request(method="rest"))
```

Comment:
I don't see the need to delete this service request, but if you must do it, you can directly put this inside a finalize block(outside the `with` block) like this - 
```
@request.addfinalizer
def _finalize():
            # Remove Request
        request_description = "Provisioning Service [{name}] from [{name}]".format(
            name=generic_catalog_item.name
        )
        service_request = appliance.collections.requests.instantiate(
            description=request_description, partial_check=True
        )
       service_request.remove_request(method="rest")
```